wkcuber/api/View.py

```diff
@@ -120,6 +122,9 @@ def for_each_chunk(self, work_on_chunk, job_args_per_chunk, chunk_size, executor
     def _check_chunk_size(self, chunk_size):
         raise NotImplementedError
 
+    def _handle_compressed_write(self, absolute_offset, data):
+        return absolute_offset, data
```

Comment:
The Dataset API does not support compressed tiffs yet (as far as I know). Therefore, I only overwrite this method for the `WKView`